README.md

```diff
@@ -32,17 +35,24 @@ services:
     volumes:
       - ./config:/usr/share/nginx/html/assets/config/
 ```
-You can change ```3000``` to a different port which you want the frontend accessible outside the container to.
 
-Inside ``` config``` folder, create a file named ```config.json``` with the following content: 
+You can change `3000` to a different port which you want the frontend accessible outside the container to.
+
+Inside ` config` folder, create a file named `config.json` with the following content:
+
 ```
 {
-    "serverEndpoint": "https://localhost:5200"
+  "serverEndpoint": "http://localhost:5200",
+  "sponsorImageUrls": ["/assets/misc/logo.webp"],
+  "sponsorImageRotateSpeed": 5000,
+  "attackerColor": "#b82e3c",
+  "defenderColor": "#3c82b8"
```

Comment:
Might be worth leaving a reference to the config.ts file here to show people an always up-to-date list of available options?

Reply:
resolved and updated to the most current values in e8bc02d26f3be8a90911377ce7eb29e55e37b4ba